graphics/sprites/monsters/terranite-mask.xml

```diff
@@ -12,7 +12,7 @@
       <frame index="1" offsetX="-1" offsetY="-40"/>
     </animation>
     <animation direction="right">
-      <frame index="3" offsetX="-1" offsetY="-40"/>
+      <frame index="3" offsetX="1" offsetY="-40"/>
```

Comment:
I cannot merge if this adds unrelated stuff O.o

Reply:
idk what to make if i make a new pull request its always in its even a different branch.